test/integration/targets/unarchive/tasks/test_different_languave_var.yml

```diff
@@ -0,0 +1,48 @@
+- name: test non-ascii with different LANGUAGE
+  block:
+    - name: install de language pack
+      apt: 
+        name: language-pack-de 
+        state: present
+    - name: Ensure de_DE.UTF-8 exists
+      community.general.locale_gen:
```

Comment:
I don't think the community.general collection is installed for tests. The test suites are just roles, you could also add a custom module to a `library/` directory in `test/integration/unarchive/` which would allow you to use it here.

Reply:
I think the locale_gen plays are actually not necessary, installing the language pack should do it. However I now get CI failures (which is obvious because apt or at least the package is missing on some systems). Can I restrict this particular test to, say, debian based clients?

Reply:
I think that's fine, I wouldn't over complicate it. We can always expand the coverage.